packages/writing-style/styles/Google/Spacing.yml

```diff
@@ -4,5 +4,5 @@ link: 'https://developers.google.com/style/sentence-spacing'
 level: error
 nonword: true
 tokens:
-  - '[.?!] {2,}[A-Z]'
-  - '[.?!][A-Z]'
+  - '[a-z][.?!] {2,}[A-Z]'
+  - '[a-z][.?!][A-Z]'
```

Comment:
Fortunately, this new change from google does not cover the `.NET` case anymore, so we don't need to override it.

Reply:
Nice trick they do here, just assuming the last letter of a sentence is normally a lowercase alphabetical one.